pennylane/math/multi_dispatch.py

```diff
@@ -870,6 +870,50 @@ def norm(tensor, like=None, **kwargs):
     return norm(tensor, **kwargs)
 
 
+@multi_dispatch(argnum=[0])
+def svd(tensor, like=None, **kwargs):

```

Comment:
Have we decided whether to have this visible in the docs? I [can't see it currently](https://xanaduai-pennylane--5154.com.readthedocs.build/en/5154/code/qml_math.html).

If we expect users to be hitting `qml.math.svd(...)` then it would be great to have this in the docs. We'd need to add to [here](https://github.com/PennyLaneAI/pennylane/blob/ba9fad8a712700dfd59cbd922d698f27c4ad1216/pennylane/math/__init__.py#L136) to get it to show up (I think).